### ISSUE TITLE
Is there a way to generate a completions bash script?

### ISSUE THREAD

**Issue by AndriiNeverov** [nodeca/argparse#149]
What would be a recommended way to achieve something similar to https://stackoverflow.com/questions/32258362/setup-command-line-completion-with-yargs?

**Comment by puzrin** [comment]
For any questions it's better to search python argparse helps. Something like `argparse tab completion`.

https://stackoverflow.com/questions/8387924/python-argparse-and-bash-completion

As far as i see, they have https://github.com/kislyuk/argcomplete, but it was not ported to JS.

**Comment by AndriiNeverov** [comment]
Yep, I used Python's argcomplete in the past and it's super powerful. Nails it with a single line of code!

Thank you for confirming. I guess this is a feature request then :)

**Comment by puzrin** [comment]
This is useful suggestion, but i have no ideas about deadlines. All i can say, In 1-2 weeks port of 3.9.4 argparse will be ready (as v2.+ o this package, see progress in branches). Then no plans to continue - too much of other deals.

**Comment by AndriiNeverov** [comment]
I see. Well, maybe some more people vote on this to bump it up the list.

In the meantime, since the API is very similar, maybe there is a workaround of making the code run in Python, letting register-python-argcomplete generate the Shell stuff from there and then re-using that.
